pkg/datapath/linux/node_addressing_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018-2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package linux
+
+import (
+	"net"
+	"testing"
+)
+
+func TestMapSubnetAddresses(t *testing.T) {
```

Comment:
We generally use gocheck for defining our unit testing suites (`import "gopkg.in/check.v1"`). It'll also provide assertions functions, etc. Is there a reason you chose to go the route of the standard `testing` instead?

Reply:
I'll refactor, but I really don't like "check". The lack of sub testing is really annoying.

Reply:
Also we do use the go testing package in many other places.